File: fix_js.js
const fs = require('fs');
let c = fs.readFileSync('View/app.js', 'utf8');
const idx = c.indexOf('handlePlanSatinAl');
console.log('Found at:', idx);
// Show 400 chars from that position
const seg = c.substring(idx, idx+400);
console.log(JSON.stringify(seg));

File: fix_js.py
import re

with open('View/app.js', 'r', encoding='utf-8') as f:
    content = f.read()

# Show what's around handlePlanSatinAl
idx = content.find('handlePlanSatinAl')
print("Found at:", idx)
print(repr(content[idx:idx+300]))
